.github/ISSUE_TEMPLATE/cherry-pick-request.yml

```diff
@@ -0,0 +1,158 @@
+---
+name: Cherry-Pick Request
+description: Request a cherry-pick to an AMP release.
+labels: ["Type: Release", "Cherry-pick: Beta", "Cherry-pick: Experimental", "Cherry-pick: LTS", "Cherry-pick: Stable"]
+title: "\U0001F338 Cherry-pick request for #ISSUE into #RELEASE"
+body:
+- type: markdown
+  id: header
+  attributes:
+    value: |
+      Thanks for filling out this cherry-pick request.
+      - See AMP's [release schedule](https://github.com/ampproject/amphtml/blob/main/docs/release-schedule.md) to learn about how releases work.
+      - See AMP's [release calendar](https://amp-release-calendar.appspot.com) for information about past releases (e.g. versions, dates, submission times, notes).
+      - See AMP's [code contribution docs](https://github.com/ampproject/amphtml/blob/main/docs/contributing-code.md#Cherry-picks) for an overview of the cherry-pick process.
+      - Use this form to provide the following details:
+        - The bug that necessitates this cherry-pick.
+        - The PR(s) that fix the bug.
+        - The release to which the cherry-pick will be applied.
+        - The channel(s) to which the cherry-pick will be applied.
+- type: input
+  id: issue
+  attributes:
+    label: Issue
+    description: The bug that necessitates this cherry-pick. Remember to update the issue title with this info.
```

Comment:
I don't think bug info is necessary on this form because you can follow the cherry-picked PR for info on that and the release engineer doesn't need it.

And sometimes the bug isn't a PR of ours but external changes i.e. browser updates.

As a release engineer, I just care about the sha to cherry-pick. Might mix up shas if there are more than 1.

Reply:
Ahh ok, I mistook "bug that necessitates this cherry-pick" as "PR that caused bug". My bad!

Reply:
On that note, can we change the description to something clearer i.e. "Issue that reports P0 bug"

Reply:
Done. Updated label and the description below it.